src/sqlfluff/cli/commands.py

```diff
@@ -1048,6 +1030,7 @@ def parse(
 
 
 def _print_out_violations_and_timing(
+    output_stream: OutputStream,
```

Comment:
Updated this function, replacing all `click.echo()` calls with `output_stream.write()`, so it can write to an output file. This is a different case from what was mentioned in the issue we're fixing, but a very similar bug -- i.e. not handling human-format output to a file.